src/sfctl/helps/chaos_schedule.py

```diff
@@ -0,0 +1,56 @@
+# -----------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# -----------------------------------------------------------------------------
+
+"""Help documentation for Service Fabric chaos-schedule commands"""
+
+from knack.help_files import helps
```

Comment:
I would put this file under helps/chaos.py, unless you think there is a good cause why this grouping should differ from Swagger specs.